packages/webapp/src/components/Task/TaskReadOnly/index.jsx

```diff
@@ -51,21 +66,27 @@ export default function PureTaskReadOnly({
 }) {
   const { t } = useTranslation();
   const taskType = task.taskType;
-  const { date, dateLabel } = useMemo(() => {
+  const { date, dateLabel, secondDate, secondDateLabel } = useMemo(() => {
```

Comment:
Yeah I agree with you there, but I was having a hard time thinking of a more meaningful name because while we only use `secondDate` and `secondDateLabel` on the abandoned tasks, they represent the due date in that scenario, which is already a case which occurs for the `date` and `dateLabel` variables if the task is not abandoned or completed. @fsayyid do you have any recommendations for a better name?

Reply:
Oh I see we are using the same variable to represent different things depending on if the task is completed or abandoned. I guess it makes sense why the naming needed to be like this then.